src/exports.jl

```diff
@@ -1400,6 +1400,7 @@ export reduced_groebner_basis
 export reduced_scheme
 export reducible_fibers
 export reflect, reflect!
+export reflection
```

Comment:
@fingolfin @simonbrandhorst I want to make you aware that this also exports the previously unexported method https://github.com/oscar-system/Oscar.jl/blob/adbd340e47b526f1d8fb99fd7712efeebc3d45d6/src/Groups/spinor_norms.jl#L116-L130

Is this fine with you (considering it has a docstring etc.) or is this still internal (aka it would be better renamed `_reflection` to not get exported by this)?